core/app/models/spree/gallery/variant_gallery.rb

```diff
@@ -0,0 +1,16 @@
+module Spree
+  module Gallery
+    class VariantGallery
+      def initialize(variant)
+        @variant = variant
+      end
+
+      # A list of all images associated with this gallery
+      #
+      # @return [Enumerable<Spree::Image>] all images in the gallery
+      def images
```

Comment:
Here as well, a `image_urls` method that takes a `size:` parameter would help us decoupling from paperclip